setup.py

```diff
@@ -21,7 +21,7 @@
     description='Genestack Python Client Library',
     long_description=__doc__,
     long_description_content_type="text/markdown",
-    install_requires=['keyring', 'requests', 'pyOpenSSL', 'jsonschema'],
+    install_requires=['keyring', 'requests', 'pyOpenSSL', 'jsonschema','pyrsistent'],
```

Comment:
I think need to add 
pyrsistent==0.16.0

Reply:
and space is likely missing after comma

Reply:
Fixed

Reply:
Sorry, but GitHub still shows the same code as before